src/components/ReportActionItem/MoneyRequestPreview.js

```diff
@@ -174,10 +174,18 @@ function MoneyRequestPreview(props) {
     // Show the merchant for IOUs and expenses only if they are custom or not related to scanning smartscan
     const shouldShowMerchant = !_.isEmpty(requestMerchant) && requestMerchant !== CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT && requestMerchant !== CONST.TRANSACTION.DEFAULT_MERCHANT;
     const shouldShowDescription = !_.isEmpty(description) && !shouldShowMerchant && !isScanning;
+    const hasPendingWaypoints = lodashGet(props.transaction, 'pendingFields.waypoints', null);
 
-    const receiptImages = hasReceipt ? [ReceiptUtils.getThumbnailAndImageURIs(props.transaction)] : [];
+    let merchantOrDescription = '';
```

Comment:
I suggest starting off this variable as `requestMerchant` rather than an empty string. That allows the removal of the `else` below.

Reply:
Agreed!

Reply:
It doesn't look like this was included in the recent changes?

Reply:
Sorry the push didn't go through, it's updated now.